docs/releasing/README.md

```diff
@@ -0,0 +1,384 @@
+
+1. Find the Dispatch and Echo L1s blockchain ID and subnet ID:
+    - [Dispath L1 details](https://subnets-test.avax.network/dispatch/details). Its subnet id is `7WtoAMPhrmh5KosDUsFL9yTcvw7YSxiKHPpdfs4JsgW47oZT5`.
+    - [Echo L1 details](https://subnets-test.avax.network/echo/details). Its subnet id is `i9gFpZQHPLcGfZaQLiwFAStddQD7iTKBpFfurPFJsXm1CkTZK`.
+1. Get the blockchain ID and VM ID of the Echo and Dispatch L1s with:
+    - Dispatch:
+
+        ```bash
+        curl -X POST --silent -H 'content-type:application/json' --data '{
+            "jsonrpc": "2.0",
+            "method": "platform.getBlockchains",
+            "params": {},
+            "id": 1
+        }'  https://api.avax-test.network/ext/bc/P | \
+        jq -r '.result.blockchains[] | select(.subnetID=="7WtoAMPhrmh5KosDUsFL9yTcvw7YSxiKHPpdfs4JsgW47oZT5") |  "\(.name)\nBlockchain id: \(.id)\nVM id: \(.vmID)\n"'
+        ```
+
+        Which as the time of this writing returns:
+
+        ```text
+        dispatch
+        Blockchain id: 2D8RG4UpSXbPbvPCAWppNJyqTG2i2CAXSkTgmTBBvs7GKNZjsY
+        VM id: mDtV8ES8wRL1j2m6Kvc1qRFAvnpq4kufhueAY1bwbzVhk336o
+        ```
+
+    - Echo:
+
+        ```bash
+        curl -X POST --silent -H 'content-type:application/json' --data '{
+            "jsonrpc": "2.0",
+            "method": "platform.getBlockchains",
+            "params": {},
+            "id": 1
+        }'  https://api.avax-test.network/ext/bc/P | \
+        jq -r '.result.blockchains[] | select(.subnetID=="i9gFpZQHPLcGfZaQLiwFAStddQD7iTKBpFfurPFJsXm1CkTZK") |  "\(.name)\nBlockchain id: \(.id)\nVM id: \(.vmID)\n"'
+        ```
+
+        Which as the time of this writing returns:
+
+        ```text
+        echo
+        Blockchain id: 98qnjenm7MBd8G2cPZoRvZrgJC33JGSAAKghsQ6eojbLCeRNp
+        VM id: meq3bv7qCMZZ69L8xZRLwyKnWp6chRwyscq8VPtHWignRQVVF
+        ```
+
+1. In the subnet-evm directory, build the VM using
+
+    ```bash
+    ./scripts/build.sh vm.bin
+    ```
+
+1. Copy the VM binary to the plugins directory, naming it with the VM ID:
+
+    ```bash
+    mkdir -p ~/.avalanchego/plugins
+    cp vm.bin ~/.avalanchego/plugins/mDtV8ES8wRL1j2m6Kvc1qRFAvnpq4kufhueAY1bwbzVhk336o
+    cp vm.bin ~/.avalanchego/plugins/meq3bv7qCMZZ69L8xZRLwyKnWp6chRwyscq8VPtHWignRQVVF
+    rm vm.bin
+    ```
+
+1. Clone [AvalancheGo](https://github.com/ava-labs/avalanchego):
+
+    ```bash
+    git clone git@github.com:ava-labs/avalanchego.git
+    ```
+
+1. Build AvalancheGo using those VM ids:
+
+    ```bash
+    cd avalanchego
+    ./scripts/build.sh ~/.avalanchego/plugins/mDtV8ES8wRL1j2m6Kvc1qRFAvnpq4kufhueAY1bwbzVhk336o
+    ./scripts/build.sh ~/.avalanchego/plugins/meq3bv7qCMZZ69L8xZRLwyKnWp6chRwyscq8VPtHWignRQVVF
```

Comment:
avalanchego does not need (and does not support) taking binaries for the build. They're very separate binaries.

Reply:
also avalanchego is/should be built into a single binary.

Reply:
Should I then just build it with `./scripts/build.sh` and avalanchego would pick them up from ` ~/.avalanchego/plugins/*` automagically?